### PR TITLE
Hexagon-tools 8.0.06 and later are dependent on 2 additional symbols:

### PR DESCRIPTION
**Pull request by ronlieb**
__cxa_finalize and __cxa_atexit
for backward compatability with hexagon_remote runtimes, we are
providing weak symbol definitions of the these functions.

### CODE REVIEW

File: src/runtime/hexagon_remote/c11_stubs.cpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
+extern "C" {
+
+// Hexagon-tools 8.0.06 and later are dependent on 2 additional symbols:
+//    __cxa_finalize and __cxa_atexit
+// for backward compatability with hexagon_remote runtimes, we are
```
Comment (dsharletg) [inline src/runtime/hexagon_remote/c11_stubs.cpp]: It's for compatibility with older toolchains/libraries, right? This *is* the runtime, so compatibility with the runtime doesn't quite make sense.

File: src/runtime/hexagon_remote/c11_stubs.cpp [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+// providing weak symbol definitions of the these functions.
+
+#include "HAP_farf.h"
+#include "HAP_power.h"
```
Comment (dsharletg) [inline src/runtime/hexagon_remote/c11_stubs.cpp]: The power header isn't needed is it?

**Comment by ronlieb** [comment]
Easier to just push a new PR with the change reverted.

**Comment by ronlieb** [comment]
Created newer PR, simpler…

https://github.com/halide/Halide/pull/1570

**Pull request closed without merging**